vbuild/apkbuild.py

```
@@ -27,6 +36,85 @@ def wrapper(self: "APKBUILD") -> str | None:
+    size = len(value)
+    while True:
+        if offset >= size:
+            if "'" in quoted_value:
+                quoted_value += "'"
+
+            break
+
+        token = value[offset]
+        offset += 1
+
+        if token != "$":
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to: "token"

(S105)